### DISCUSSION TITLE
prebuilt Singularity image for Topaz has not been updated

### DISCUSSION THREAD

**Discussion by refolding** [tbepler/topaz#161] [General]
The current prebuilt Singularity image for Topaz is still at v 0.2.4, will you please update it to v 0.2.5? or is there a way to update the image by the user? 

Thanks!

**Comment by refolding** [comment]
After doing some research, indeed one can update the Singularity image:
#build a sandbox from the image file
sudo singularity build --sandbox topaz_update/ topaz_latest.sif 
#shell into the sandbox
sudo singularity shell --writable topaz_update/
#inside the sandbox, update topaz:
/usr/local/conda/bin/conda install  -y topaz cuda80 -c tbepler -c soumith
#exit sandbox shell, rebuild Singularity image:
sudo singularity build topaz_update.sif topaz_update/

**Reply by eMKiso** [reply]
Hi @refolding,

would you be willing to share your updated topaz_latest.sif?

Best!

**Reply by refolding** [reply]
Sure I will email you the download link, but you must have a Box account to download it. It should be free to register.

**Reply by refolding** [reply]
Can I have your email?

**Comment by eMKiso** [comment]
Hi,

I opened an Issue ticket to update the Singularity image but no response yet.
Ideally the authors would update it but one can do it manually as @refolding  suggested.
